types/v56/types.go

```diff
@@ -758,6 +758,28 @@ type Catalog struct {
 	VersionNumber int64            `xml:"VersionNumber"`
 }
 
+// AdminCatalog represents the Admin view of a Catalog object.
```

Comment:
Compared to `Catalog`, the only differences that I see are:
* `AdminCatalog` has `XMLName`  and `Xmlns` and `Catalog` doesn't (but it can have both because the API definition contains them)
* `Catalog` has `DateCreated` and `AdminCatalog` doesn't (but it should).
In short, the two structures are identical and should share the definition.

OTOH, the [API entry for AdminCatalog](https://code.vmware.com/apis/287/vcloud#/doc/doc/types/AdminCatalogType.html) defines several elements that are not in this structure, such as `CatalogStorageProfiles`, `ExternalCatalogSubscriptionParams`, `PublishExternalCatalogParams`. 
I think this definition needs more analysis.

Reply:
you are right, and I am going to add those elements to make the definition complete